client/src/modules/user/UserProfile/UserProfile.jsx

```diff
@@ -202,11 +184,14 @@ class UserProfile extends PureComponent {
 
   render() {
     const { pending } = this.state;
+    const {
+      currentUser: { avatarUrl, name }
+    } = this.props;
 
     return (
       <div className="wrapper">
         <article className="user-profile">
-          {this.renderHeader()}
+          <UserProfileHeader avatarUrl={avatarUrl} name={name} />
           {this.renderPersonalInfo()}
```

Comment:
What about the others?

Reply:
Added as a separated task.
https://github.com/sandstreamdev/eoc/pull/275#discussion_r314635898